### ISSUE TITLE
Add alt text in search icon

### ISSUE THREAD

**Issue by kburk1997** [YACS-RCOS/yacs-web#83]
I ran YACS through http://wave.webaim.org and we are missing alt text for the search icon. 

Add alt text to the search icon so that screenreader users can understand the search bar.

**Comment by sajeetharan** [comment]
@kburk1997  where can i find the search icon?
